template/electron/index.js

```diff
@@ -1 +1,78 @@
-require('./main');
+// Import parts of electron to use
+const {app, BrowserWindow} = require('electron');
+const path = require('path')
+const url = require('url')
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let mainWindow;
+
+// Keep a reference for dev mode
+let dev = false;
+if ( process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath) ) {
```

Comment:
I don't think it hurts, was more of a safety check given I didn't have that information available to me at the time.